src/com/palmergames/bukkit/towny/Towny.java

```diff
@@ -184,6 +184,15 @@ public String call() throws Exception {
 					townyUniverse.onLogin(player);
 				}
 		}
+		
+		IntegerDataField level = new IntegerDataField("level", 0);
```

Comment:
Might want to comment the reason as to why this is here. Otherwise maintainers will just have to guess the intention.

Reply:
Thats an artifact of my prototyping I'll make sure to remove it